<ui version="4.0" stdsetdef="1" >
  <class>QSqlConnectionDialogUi</class>
  <widget class="QDialog" name="connectionDialog" >
    <property name="objectName" >
      <string notr="true" >connectionDialog</string>
    </property>
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>303</width>
        <height>247</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Connect...</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="objectName" >
        <string notr="true" >unnamed</string>
      </property>
      <item>
        <widget class="QGroupBox" name="GroupBox" >
          <property name="objectName" >
            <string notr="true" >GroupBox3</string>
          </property>
          <property name="title" >
            <string>Connection settings</string>
          </property>
          <layout class="QGridLayout" >
            <property name="objectName" >
              <string notr="true" >unnamed</string>
            </property>
            <item row="1" column="1" >
              <widget class="QLineEdit" name="editDatabase" >
                <property name="objectName" >
                  <string notr="true" >editDatabase</string>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QLabel" name="textLabel3" >
                <property name="objectName" >
                  <string notr="true" >textLabel3</string>
                </property>
                <property name="text" >
                  <string>Database Name:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>editDatabase</cstring>
                </property>
              </widget>
            </item>
            <item row="3" column="1" >
              <widget class="QLineEdit" name="editPassword" >
                <property name="objectName" >
                  <string notr="true" >editPassword</string>
                </property>
                <property name="echoMode" >
                  <enum>QLineEdit::Password</enum>
                </property>
              </widget>
            </item>
            <item row="2" column="0" >
              <widget class="QLabel" name="textLabel4" >
                <property name="objectName" >
                  <string notr="true" >textLabel4</string>
                </property>
                <property name="text" >
                  <string>&amp;Username:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>editUsername</cstring>
                </property>
              </widget>
            </item>
            <item row="3" column="0" >
              <widget class="QLabel" name="textLabel4_2" >
                <property name="objectName" >
                  <string notr="true" >textLabel4_2</string>
                </property>
                <property name="text" >
                  <string>&amp;Password:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>editPassword</cstring>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QComboBox" name="comboDriver" >
                <property name="objectName" >
                  <string notr="true" >comboDriver</string>
                </property>
              </widget>
            </item>
            <item row="2" column="1" >
              <widget class="QLineEdit" name="editUsername" >
                <property name="objectName" >
                  <string notr="true" >editUsername</string>
                </property>
              </widget>
            </item>
            <item row="4" column="1" >
              <widget class="QLineEdit" name="editHostname" >
                <property name="objectName" >
                  <string notr="true" >editHostname</string>
                </property>
              </widget>
            </item>
            <item row="4" column="0" >
              <widget class="QLabel" name="textLabel5" >
                <property name="objectName" >
                  <string notr="true" >textLabel5</string>
                </property>
                <property name="text" >
                  <string>&amp;Hostname:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>editHostname</cstring>
                </property>
              </widget>
            </item>
            <item row="5" column="0" >
              <widget class="QLabel" name="textLabel5_2" >
                <property name="objectName" >
                  <string notr="true" >textLabel5_2</string>
                </property>
                <property name="text" >
                  <string>P&amp;ort:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>portSpinBox</cstring>
                </property>
              </widget>
            </item>
            <item row="5" column="1" >
              <widget class="QSpinBox" name="portSpinBox" >
                <property name="objectName" >
                  <string notr="true" >portSpinBox</string>
                </property>
                <property name="specialValueText" >
                  <string>Default</string>
                </property>
                <property name="maximum" >
                  <number>65535</number>
                </property>
                <property name="minimum" >
                  <number>-1</number>
                </property>
                <property name="value" >
                  <number>-1</number>
                </property>
              </widget>
            </item>
            <item row="0" column="0" >
              <widget class="QLabel" name="textLabel2" >
                <property name="objectName" >
                  <string notr="true" >textLabel2</string>
                </property>
                <property name="text" >
                  <string>D&amp;river</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>comboDriver</cstring>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <item>
            <spacer name="Spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okButton" >
              <property name="objectName" >
                <string notr="true" >okButton</string>
              </property>
              <property name="text" >
                <string>&amp;OK</string>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton" >
              <property name="objectName" >
                <string notr="true" >cancelButton</string>
              </property>
              <property name="text" >
                <string>&amp;Cancel</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <tabstops>
    <tabstop>comboDriver</tabstop>
    <tabstop>editDatabase</tabstop>
    <tabstop>editUsername</tabstop>
    <tabstop>editPassword</tabstop>
    <tabstop>editHostname</tabstop>
    <tabstop>portSpinBox</tabstop>
    <tabstop>okButton</tabstop>
    <tabstop>cancelButton</tabstop>
  </tabstops>
</ui>
